src/main/java/Lottery.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int maxNumber = 100;
```

Comment:
constant are private static final and has UPPERCASE_NAMING
also it's better to place them before other variables
```suggestion
    private static final int MAX_NUMBER = 100;

    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```